category.php

```diff
@@ -47,7 +47,7 @@
 <?php
 }
 
-$user = isset($_REQUEST["user"]) ? $_REQUEST["user"] : NULL;
+$user = NULL; // Editing as user or script, no need to name.
 if (is_valid_user($user)) {
   echo " Activated by $user.\n\n";
   $edit_summary_end = " | [[User:$user|$user]]; [[Category:$category]].";
```

Comment:
This is dead code, and, once the edits are done using OAUTH, there is no need to add "user-activated" to the edit summary (alternately, replace it with something like "using Citation bot")

Reply:
Original attempt was to minimize code changes